client/wc-api/wp-data-store/create-api-client.js

```diff
@@ -19,11 +19,6 @@ function createStore( name ) {
 function createDataHandlers( store ) {
 	return {
 		dataRequested: resourceNames => {
-			const { resources } = store.getState();
-			const newResources = resourceNames.some( resourceName => ! resources[ resourceName ] );
-			if ( ! newResources && document.hidden ) {
```

Comment:
After removing this, I see `FRESH_DATA_REQUESTED` is being dispatched even when the tab is inactive. That means we update the [`lastRequested`](https://github.com/woocommerce/woocommerce-admin/blob/master/client/wc-api/wp-data-store/reducer.js#L26) value even though no request is made.

That's an issue in the following scenario. Imagine I have two tabs open and WooCommerce Admin is an inactive tab. If I reload it without activating, `FRESH_DATA_REQUESTED` is dispatched (so `lastRequested` updated). Then, switching to WooCommerce Admin will show no data and it will not update it until it timeouts:
![Peek 2019-06-11 10-58](https://user-images.githubusercontent.com/3616980/59258212-ceac5080-8c37-11e9-8078-ba0a5f121347.gif)

Can you reproduce that too?

Reply:
I'm able to reproduce this @Aljullu.

Reply:
Adding this back in resolved for me:

```javascript
if ( document.hidden ) {
    return;
}
```

Reply:
That makes sense. I'll see what I can do to adjust this.

Ultimately, I think this will be a temporary fix. I'm working on an update to fresh-data now and I can incorporate a better fix within that.

Reply:
Can we add this temporarily with a comment? It would be nice to get this out asap in `0.13.0` today or tomorrow.

Reply:
Done in 26cf428.

Reply:
I confirmed this solves the issue. As soon as you switch to activate the wc-admin page, a request is made.